src/main/java/roman/RomanToNumberToRoman.java

```diff
@@ -46,4 +36,65 @@ private int getIntValueOfCurrentRoman(char currentRoman) {
 		}
 		return currentNumber;
 	}
-}
+
+	public String toRoman(int number) {
+		StringBuilder sb = new StringBuilder();
+		char currentNumeral = ' ';
+		int powerOf = 0;
+		int baseNumber = 0;
+		int numberOfRomans = 0;
+		while (number > 9) {
+			powerOf = (int) (Math.log10(number));
+			baseNumber = (int) (Math.pow(10, powerOf));
+			int newBaseNumber = 0;
+			numberOfRomans = number / baseNumber;
+			while (numberOfRomans > 0) {
+				if (numberOfRomans == 4) {
```

Comment:
What's special about `4` here? Consider pulling that out to a constant to explain its significance.